lego/apps/banners/tests.py

```diff
@@ -0,0 +1,133 @@
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from lego.apps.banners.models import Banners
+
+User = get_user_model()
+
+
+def get_banner_list_url():
+    return reverse("api:v1:banner-list")
+
+
+def get_banner_detail_url(pk):
+    return reverse("api:v1:banner-detail", kwargs={"pk": pk})
+
+
+def get_current_public_url():
+    return reverse("api:v1:banner-current-public")
+
+
+def get_current_private_url():
+    return reverse("api:v1:banner-current-private")
+
+
+class BannersPermissionTestCase(APITestCase):
+    # Import the fixture containing the Abakus groups.
+    fixtures = ["test_abakus_groups.yaml"]
+
+    def setUp(self):
+        # Create a test user.
+        self.user = User.objects.create_user(username="testuser", password="pass")
+        # Get the admin group from the fixture (adjust the group name if needed).
+        from lego.apps.users.models import AbakusGroup
+
+        self.admin_group = AbakusGroup.objects.get(name="Webkom")
+        self.default_data = {
+            "header": "Test Banner",
+            "subheader": "Test Subheader",
+            "link": "http://example.com",
+            "color": "red",  # assuming "red" is a valid option
+            "current_public": False,
+            "current_private": False,
+        }
+
+    def test_banner_create_without_permission(self):
+        """
```

Comment:
Maybe ensure the other default endpoints also require permission since it seems `current_private` and `current_public` are the only ones that should ever be available for non-admins?